pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -2350,3 +2350,21 @@ query B
 SELECT (1, 2) IN ((2, 3), NULL)
 ----
 NULL
+
+# Test for regression in hex functions.
+subtest regression_41707
+
+query TTTTTTT
+select to_hex(-2147483649), to_hex(-2147483648), to_hex(-1), to_hex(0), to_hex(1), to_hex(2147483647), to_hex(2147483648)
```

Comment:
Add type casts to all of the numbers so that when you run this query the output is identical to postgres.